src/entries/background/index.ts

```diff
@@ -18,6 +19,7 @@ import { handleWallets } from './handlers/handleWallets';
 require('../../core/utils/lockdown');
 
 initializeSentry('background');
+await localStorageRecycler();
```

Comment:
```suggestion
localStorageRecycler();
```

There's no reason to block here, this can happen in paralell